packages/api/src/API/Http/Actions/Category/EditCategoryAction.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express';
+import TYPES from '../../../../Infraestructure/types';
+import CategoryEditHandlerInterface from '../../../../Infraestructure/Interfaces/Category/CategoryEditHandlerInterface';
+import { inject, injectable } from 'inversify';
+import EditProductAdapter from '../../Adapter/Category/EditCategoryAdapter';
+import CategoryEditCommand from '../../../../Application/Commands/Category/CategoryEditCommand';
+
+@injectable()
+class EditCategoryAction{
+    
+    private handler: CategoryEditHandlerInterface;
+    private adapter: EditProductAdapter;
+
+    constructor(
+        @inject(TYPES.ICategoryEditHandler) handler: CategoryEditHandlerInterface,
+        @inject(EditProductAdapter) adapter: EditProductAdapter,
+    ){
+        this.handler = handler;
+        this.adapter = adapter;
+    }
+
+
+    public async execute(req: Request, res: Response) {
+        const command: CategoryEditCommand = await this.adapter.from(req);
+        const response: string = await this.handler.Handle(command);
```

Comment:
Tanto el Create and Edit deberían devolver la entidad por convención, pero lo dejo como detalle

Reply:
Cris! a ver si entendi, esta correccion que hay que hacer es con la aplicacion de los presenters, ose aca es donde le voy a preparar los datos en JSON para que se comunique con website (front) - si estoy en lo correcto, esta correccion la hago en la issue #60

Reply:
Si si, eso tiene que ver con el presenter, pero como dije acá lo dejo como detalle ya que eso corresponde a otra issue